machine/charmcraft.yaml

```diff
@@ -0,0 +1,173 @@
+# Copyright 2024 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+name: vault
+
+summary: A tool for managing secrets
+description: |
+  Vault secures, stores, and tightly controls access to
+  tokens, passwords, certificates, API keys, and other
+  secrets in modern computing. Vault handles leasing, key
+  revocation, key rolling, and auditing. Through a unified
+  API, users can access an encrypted Key/Value store and
+  network encryption-as-a-service, or generate AWS IAM/STS
+  credentials, SQL/NoSQL databases, X.509 certificates,
+  SSH credentials, and more.
+links:
+  website:
+    - https://charmhub.io/vault
+  source:
+    - https://github.com/canonical/vault-operator
+  issues:
+    - https://github.com/canonical/vault-operator/issues
```

Comment:
This will probably need to change also, but maybe we can wait until this is merged and we rename the repository. WDYT?

Reply:
Good catch, done here.